hypermap/aggregator/solr.py

```diff
@@ -0,0 +1,233 @@
+        solr_url_parts = solr_url.split('/')
+        admin_url = '/'.join(solr_url_parts[:-1]) + '/admin/cores'
+        params = {'action': 'STATUS', 'wt': 'json'}
+        try:
+            req = requests.get(admin_url, params=params)
+            response = json.loads(req.text)
+            status = response['status']
+            if status:
+                response = True
+        except requests.exceptions.RequestException:
+            response = False
+        return response
+
+    @staticmethod
+    def layer_to_solr(layer, i=0):
+        try:
+            bbox = [float(layer.bbox_x0), float(layer.bbox_y0), float(layer.bbox_x1), float(layer.bbox_y1)]
+            for proj in layer.srs.values():
+                if '102113' == proj['code']:
+                    bbox = mercator_to_llbbox(bbox)
+            # to do need to figure out it hypermap will only be dealing with remote servives
+            storeType = "remoteStore"
+            # We will need to get the layer date lists if they exist here and use this instead as a date range
+            date = layer.created
+            if (Solr_Hypermap.good_coords(bbox)) is False:
+                print 'no coords in layer ', layer.title
+                return
+            if (Solr_Hypermap.good_coords(bbox)):
+                print 'in solr.layer_to_solr, bbox = ', bbox
+                username = ""
+                minX = bbox[0]
+                minY = bbox[1]
+                maxX = bbox[2]
+                maxY = bbox[3]
+                if (minY > maxY):
+                    tmp = minY
+                    minY = maxY
+                    maxY = tmp
+                if (minX > maxX):
+                    tmp = minX
+                    minX = maxX
+                    maxX = tmp
+                centerY = (maxY + minY) / 2.0
+                centerX = (maxX + minX) / 2.0
+                halfWidth = (maxX - minX) / 2.0
+                halfHeight = (maxY - minY) / 2.0
+                area = (halfWidth * 2) * (halfHeight * 2)
+                if (minX < -180):
+                    minX = -180
+                if (maxX > 180):
+                    maxX = 180
+                if (minY < -90):
+                    minY = -90
+                if (maxY > 90):
+                    maxY = 90
+                # ENVELOPE(minX, maxX, maxY, minY) per https://github.com/spatial4j/spatial4j/issues/36
+                wkt = "ENVELOPE({:f},{:f},{:f},{:f})".format(minX, maxX, maxY, minY)
+                # default for now as we do not have capabilitty to resolve vector/raster layers
+                dataType = "Polygon"
+                institution = "Harvard"
+                servicetype = layer.service.type
+                owsUrl = layer.service.url
+                if storeType == "remoteStore":
+                    institution = "Remote"
+                    if servicetype == "ESRI":
+                        dataType = "RESTServices"
```

Comment:
why serviceteype and dataType are named differently?
and why their content is different?